_posts/en/pages/2016-01-18-segwit-wallet-dev.md

```diff
@@ -0,0 +1,124 @@
+---
+version: 1
+title: Segregated Witness Wallet Development Guide
+name: segwit-wallet-dev
+type: page
+permalink: /en/segwit_wallet_dev
+---
+
+### Witness program
```

Comment:
Also, there should be links to BIP141, BIP142, BIP143, etc, and the other relevant BIPs. Thanks.